src/requireUniversalModule.js

```diff
@@ -51,7 +51,7 @@ export default function requireUniversalModule<Props: Props>(
   const requireSync = (props: Object, context: Object): ?any => {
     let exp = loadFromCache(chunkName, props, modCache)
 
-    if (!exp) {
+    if (!exp || isServer) {
```

Comment:
tuck it in `loadFromCache`, as `requireAsync` may need it too.

Also, additional formatting was added to the code that should not have been.